M2/Macaulay2/m2/rationals.m2

```diff
@@ -48,9 +48,9 @@ QQ.Engine = true
 assert (hash ZZ < hash QQ)
 
 lift(QQ,ZZ) := opts -> (r,o) -> if denominator r === 1 then numerator r else if opts.Verify then error "rational number is not an integer"
-liftable(QQ,ZZ) := (r,o) -> denominator r === 1
+isLiftable(QQ,ZZ) := (r,o) -> denominator r === 1
 lift(QQ,QQ) := opts -> (r,QQ) -> r
-liftable(QQ,QQ) := (QQ,QQ) -> true
+isLiftable(QQ,QQ) := (QQ,QQ) -> true
```

Comment:
Why does this line even exist? We don't have `isLiftable(ZZ,ZZ)`, etc.. Maybe there should be `isLiftable(Ring, Ring) := (R, S) -> R === S` that is used as a last resort if R and S don't have an `isLiftable` method installed?

Reply:
you're right, this line is useless I believe.
right now for most (all?) rings we have `lift(R,R):=o->(x,R)->x` so `isLiftable(R,R)` will give `true` without any need to override the default behaviour.